web/ui/navbar/other.svelte

```diff
@@ -22,6 +22,7 @@
   href={getURL('add')}
   icon={mdiPlusCircleOutline}
   secondary
+  on:click
```

Comment:
Tell me more how does `on:click` works here.

I am not very advanced Svelte developer too.

Reply:
In Svelte component events don't bubble. So, if you want to listen to an event on a deeply nested component, the intermediate components need to forward the event.

Reply:
Nice to know, thanks